templates/projects/empty-server/data.js

```diff
@@ -79,8 +79,8 @@ template.server = {
           limit: '100kb'
         },
         cors: false,
-        errorHandler: {
-          disableStackTrace: false
+        strongErrorHandler: {
```

Comment:
I find it difficult to review changes in templates without seeing the corresponding changes in runtime (strong-remoting and/or loopback). Could you please point me to a pull request adding `strongErrorHandler` config option to strong-remoting?